City-of-Helsinki/drupal-helfi-etusivu
# UHF-10772: Add events list to Helsinki near you

# [UHF-10772](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10772)
Continues work on the Helsinki near you -feature.

## What was done
* Add template that embeds linkedevents app
* Add new variables to drupalSettings
* Add translations

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10772`
  * `make fresh`
* Get the HDBT feature branch: `composer require drupal/hdbt:dev-UHF-10772`
* Run `make drush-cr`
* Update translations: `make drush-locale-update`

## How to test

* Check the sketch: https://www.figma.com/design/udh2zu4J95VYDIzT7DtbXW/Alueellisuus
* Go to Near you results page, eg. https://helfi-etusivu.docker.so/fi/helsinki-lahellasi/tulokset?q=Vaasankatu%2B4
* Check that the results you are gettings are located ~2km away from the address you input
* Evaluate these styles and functionalities especially:
  * New "ghost" cards (pre-load placeholders)
  * New loading animation for the cards
  * New "bordered" card style
* Test these basic functionalities
  * Pagination works
  * External linking for the events works
  * Translations work for all languages

Bonus points: 
* Test that the events paragraph still works as intended on a landing page

## Continuous documentation



## Translations



## Other PRs


[HDBT](https://github.com/City-of-Helsinki/drupal-hdbt/pull/1131)

[UHF-10772]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
